includes/namespace.php

```diff
@@ -26,6 +26,8 @@
 
 namespace Google\Web_Stories;
 
+global $heading, $body;
```

Comment:
I really hate the use of globals here. Why not just a function, have functions like `_print_missing_build_header` and `_print_missing_build_body` that just return the strings.

Reply:
Fair enough, although I dislike adding two new functions specifically for this.

I will just copy/repeat the message. Adds some repetition, but is way easier to maintain.

Reply:
Why not just one function that returns an array, with `header` and `body` keys? These being globals, also opens you up to someone overriding them in another piece of code and a possible security risk.

Reply:
I don't disagree with the globals part.

But I think we don't need an extra function either.

Reply:
Okay, I agree.